python/cudf/cudf/_lib/pylibcudf/groupby.pyx

```diff
@@ -90,14 +90,39 @@ cdef class GroupBy:
         Whether or not to include null rows in ``keys``. Default is null_policy.EXCLUDE.
     keys_are_sorted : sorted, optional
         Whether the keys are already sorted. Default is sorted.NO.
+    column_order : list[order]
+        Order of each key column if the keys are sorted. Default if not
+        provided uses order.ASCENDING. Ignored if keys_are_sorted is sorted.NO.
+    null_precedence : list[null_order]
+        Where do nulls sort if the keys are sorted?
```

Comment:
I've always felt like it helps maintain pylibcudf's faithfulness to libcudf to just copy most of the parameter descriptions wholesale from libcudf's [docs on groupby](https://docs.rapids.ai/api/libcudf/legacy/classcudf_1_1groupby_1_1groupby), which is also less work.

Reply:
Reworded a bit.